source/upgrade.txt

```diff
@@ -116,6 +116,20 @@ Version 5.0 Breaking Changes
   .. After the 5.0 Scala API docs are released, this line will be uncommented. 
     For more information, see the `Observable trait documentation <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__.
 
+- This driver version removes the following classes from the
+  ``com.mongodb.event`` package:
```

Comment:
S: [My PR](https://github.com/mongodb/docs-java/pull/510/files#diff-9c988395fdf1141e52aa9fa946e169ad1d9fb7de62fd44fc1b95bc1885b9506dR64) hasn't been merged yet, but it changes the language used to introduce these bullet points. We can either leave this bullet point as is for now and come back later to reword, or change it as part of this ticket to match the changes in my PR.

Reply:
Ya, there's going to be a few changes required to several PRs - once yours is merged, we can start rebasing the other ones, and account for those changes.